benchmarks/helics/EchoMessageHubFederateMain.cpp

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Would there be any sense in just having a single main with an argument which one to run, since a lot of these main operations look pretty similar now and have a common base class?

Reply:
Yea, merging all of these into a single executable with subcommands is a good idea -- it would be a nice way to get rid of all the extra files.